redbot/cogs/mod/kickban.py

```diff
@@ -173,6 +173,17 @@ class KickBanMixin(MixinMeta):
             await ctx.send(_("I cannot do that due to discord hierarchy rules"))
             return
         audit_reason = get_audit_reason(author, reason)
+        if reason is None:
+            reason = "No reason was given."
```

Comment:
This should be wrapped in `_()` for localization.
```suggestion
            reason = _("No reason was given.")
```